src/main/resources/META-INF/plugin.xml

```diff
@@ -10,12 +10,12 @@
     ]]></description>
 
     <change-notes><![CDATA[
-    1.0.12 - Add support for 221* / 2022.1 version of IDEA
+    1.0.13- Add trailing comma support
     ]]>
     </change-notes>
 
     <!-- please see https://confluence.jetbrains.com/display/IDEADEV/Build+Number+Ranges for description -->
-    <idea-version since-build="202" until-build="213.*"/>
+    <idea-version since-build="191.8026.42" until-build="221.*"/>
```

Comment:
Aligned to actual values (may consider removing this from here or from the build.gradle, to make it easier to keep up to date)

Reply:
agree. happy to remove these 👍